test/capi/capiText.cpp

```diff
@@ -69,7 +69,7 @@ TEST_CASE("Load/unload TTF file from a memory", "[capiText]")
     fseek(file, 0, SEEK_SET);
     char* data = (char*)malloc(data_size);
     REQUIRE(data);
-    REQUIRE(fread(data, 1, data_size, file) == data_size);
```

Comment:
why don't data type casting?

Reply:
the only reason is because in all other places we use this kind of check (>0)